manifests/base/files/crd-operator.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why not generate this like all of our other CRDs?

Reply:
Basically it's because the resulting OpenAPI schema is too huge.